nrf_modem/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,53 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+nrf_modem
+*********
+
+Core library
+============
+
+* Fixed a bug introduced in the :c:func:`nrf_modem_init()` function in version 2.3.0, where the library would use the function's input parameter after the function had returned.
+  This could cause several socket functions to return an error an set ``errno`` to ``NRF_EINVAL``.
```

Comment:
```Suggestion
This could cause several socket functions to return an error and set ``errno`` to ``NRF_EINVAL``.
```

Reply:
I think we want to have it indented like this as it is part of the above bullet point.

Reply:
Yes, have updated suggestion.

Reply:
This is already fixed.